app/konnect/analytics/use-cases.md

```diff
@@ -0,0 +1,190 @@
+This can be a good measure of the user experience when a user makes a request to an API.
+
+## Build reports 
+
+Let's build some custom reports by navigating to {% konnect_icon analytics %} **Analytics** in the {{site.konnect_short_name}} menu, then **Reports**.
+
+This brings you to a list of all custom reports in the organization. From here, click **New Report** to get started.
+
+### Total number of requests across APIs over the last 30 days
+
+To build a report that displays the total number of requests across all of your APIs over the last 30 days, set the following options in the UI: 
+
+* **Name**:  API usage (last 30 days)
+* **Horizontal Bar Chart**
+* **Date/Time**: Last 30 days
+* **Select a metric**: Request count
+* **Group by**: Service
+* **Then by**: None
+
+![API usage (last 30 days)](/assets/images/docs/konnect/custom-reports/total-api-requests.png){:.image-border}
+
+This report can provide your stakeholders with the answers to questions like:
+
+* How many requests were made to the API during the current month? 
+* How does this month compare to the previous months? 
+* Which service receives the most requests?
+
+The data can be used to support other types of exploratory questions about your API: 
+
+* What is the most popular service? 
+* Are there spikes in requests across months? 
+* Are there any errors or issues that might be impacting the number of requests our API is receiving? 
+
+### Daily API usage for an API over the last 30 days
+
+From the previous report, you determine that the Accounts API is receiving the most traffic. 
+You don't know whether this is a cause for concern or not, so you decide to take a closer look.
+
+To gain further insight into the Accounts API, let's build a report that displays the number of requests per day for the Accounts API over the last 30 days. 
+Set the following options in the UI:
+
+* **Name**: Daily Accounts API Usage (last 30 days)
+* **Chart type**: Line chart
+* **Date/Time**: Last 30 days
+* **Select a metric**: Request count
+* **Group by**: Service
+* **Choose granularity**: Daily
+
+Add a filter for the Accounts API. Click **Add Filter**, then set the following options:
+
+* **Filter by**: Service
+* **Operator**: In
+* **Filter value**: Accounts API (or your own API name)
+
+![Daily Accounts API Usage (last 30 days)](/assets/images/docs/konnect/custom-reports/total-usage-accounts-api-30.png){:.image-border}
+
+Now that you know what day or days this spike happened on, let's dig a little deeper. 
+Edit the report to switch the metric from **Request Count** to **Requests per Minute**.
+
+Your configuration should now look like this, with the filter unchanged:
+
+* **Date/Time**: Last 30 days
+* **Select a metric**: Requests Per Minute
+* **Group by**: Service
+* **Choose granularity**: Daily
+
+![Daily Accounts API Usage (last 30 days)](/assets/images/docs/konnect/custom-reports/total-api-usage-by-application-minute.png){:.image-border}
```

Comment:
Same feedback as in the previous comment. The filter badge is not correct.

Reply:
Also, the metric selected in the image is "Request Count" but it should be "Request per Minute".